toc.md

```diff
@@ -811,34 +818,33 @@ def expect_during(duration_in_secs, target)
 end
 ```
 
-In order to get around the asynchronous nature of threads, instead of asserting the final state of the system, we expect a certain state of the system to be mutated within a specified timeframe.
-
 ### Benchmarking Workerholic
 #### Workerholic compared to the Gold Standard: Sidekiq
-![benchmark_workerholic_sidekiq](/images/benchmark_workerholic_sidekiq.png)
 
 Finally, we wanted to compare with Sidekiq one last time with each types of jobs individually. We're on par with Sidekiq, and only slightly faster than Sidekiq each time, and as we mentioned before this is because Sidekiq is a more mature and robust solution with a much larger feature-set.
 
-#### JRuby
-![benchmark_jruby](/images/benchmark_jruby.png)
+![benchmark_workerholic_sidekiq](/images/benchmark_workerholic_sidekiq.png)
 
 We also decided to compare the results of JRuby vs MRI. Because jRuby can run in parallel without the need of spinning up multiple processes, we found that execution of CPU blocking jobs was much faster in JRuby than in MRI, which is what we would expect.
 
+#### JRuby
+![benchmark_jruby](/images/benchmark_jruby.png)
+
 ## Conclusion
 
-It was a lot of fun to work on this project! Our goal was to share potentially useful knowledge with the community and to introduce background job processors to people unfamiliar with them. After all it's usually a very interesting enterprise to understand how something works under the hood!
+It was a lot of fun to work on this project! Our goal was to share this knowledge with the community and. We are confident our experience will be useful to other software engineers even if they don't have a plan to developer their own background job processor. After all it's usually a very interesting enterprise to understand how something works under the hood!
```

Comment:
I found your typos when you were fixing mine lol

"with the community and. We are"